mergify_engine/worker.py

```diff
@@ -557,6 +557,7 @@ async def _consume_buckets(
             )
 
             messages = await self.redis_stream.xrange(bucket_sources_key)
+            statsd.histogram("engine.buckets.read_size", len(messages))  # type: ignore[no-untyped-call]
```

Comment:
it's the same as engine.streams.batch-size

Reply:
Maybe we should drop engine.streams.batch-size and keep your's !

Reply:
> Actually I think `engine.streams.batch-size` is the same, no?

The only diff is `engine.streams.batch-size` is not computed for pull number 0

Reply:
Good point, I've changed it that way.